docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  tlklan:
+    image: tlk/tlklan:latest
```

Comment:
Nitpicking here, but the Github organization is named `tlklan` so maybe `tlklan/tlklan` would be better?